### DISCUSSION TITLE
Do not update update password field if empty

### DISCUSSION THREAD

**Discussion by on3ss** [filamentphp/filament#2723] [Help] [answered]
I have a form where I have a password field that is required on create but optional on edit and it shouldn't be updated to the database if left empty. How do I do this??

**Comment by danharrin** [comment, marked as answer]
https://filamentphp.com/tricks/password-form-fields

**Reply by on3ss** [reply]
Thank you

**Reply by obrunsmann** [reply]
@danharrin is there any chance to not prefill the field with the hashed password? 🧐 

The trick works perfectly fine, the only thing I miss is the empty field 😇 

<img width="971" alt="image" src="https://user-images.githubusercontent.com/763812/184839021-98b2840b-3939-44d5-a84c-8933750100ef.png">

**Reply by danharrin** [reply]
Make sure your model's `$hidden` contains `password`

**Reply by obrunsmann** [reply]
> Make sure your model's `$hidden` contains `password`

Oh yeah 🥳 Thank you!!
